Examples/Examples/ExamplesApp.swift

```diff
@@ -19,6 +19,6 @@ struct ExamplesApp: App {
 }
 
 let supabase = SupabaseClient(
-  supabaseURL: Secrets.supabaseURL,
+  supabaseURL: Secrets.supabaseURL.absoluteString,
```

Comment:
With the new convenience methods we ended up with everything being a string in the interface, would you like to keep this as a URL? If so I can edit the tests (I think they are passing in strings)

Reply:
Let's keep as a URL

Reply:
I ended up removing the convenience initializers since their signatures conflicted with the default initializer, but also I feel like it's better to force people to specify this auth option on all platforms since we can't guess what people are building exactly. 

It's entirely possible to be writing code on macOS for a codebase that's going to be used on Linux. This would cause confusion for consumers of this package making the assumption that since they didn't make a specific choice about the secure storage engine. 

I realize this is quite opinionated, and I can probably provide some static functions or something like that to configure 'default' clients, but it felt worth checking this assumption since it can easily create confusion in a cross platform codebase.